src/Route.jsx

```diff
@@ -13,10 +17,27 @@ const STOP_ICON_MAPPING = {
 
 const BUS_ICON_MAPPING = {
   marker: {
-    x: 0, y: 0, width: 128, height: 128, mask: false,
+    x: 0, y: 0, width: 128, height: 128, mask: true,
   },
 };
 
+/* https://en.wikipedia.org/wiki/Template:MUNI_color */
+const MUNI_COLOR = [
+  { line: 'E', color: [102, 102, 102] },
+  { line: 'F', color: [240, 230, 140] },
+  { line: 'J', color: [250, 166, 52] },
+  { line: 'K', color: [86, 155, 190] },
```

Comment:
There's no `K` or `T` lines at the moment (until Central subway opens), just `KT`. Could you rename this to KT?